src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/DefaultAssemblyProvider.cs

```diff
@@ -88,7 +88,7 @@ public IEnumerable<Assembly> CandidateAssemblies
         /// </summary>
         /// <returns>A set of <see cref="Library"/>.</returns>
         // Internal for unit testing
-        protected internal virtual IEnumerable<RuntimeLibrary> GetCandidateLibraries()
```

Comment:
The type is `internal` now. So we can't expose it unless we make our private implementation public.